packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -350,6 +350,10 @@ class BuildIOSArchiveCommand extends _BuildIOSSubCommand {
     if (xcodeProjectSettingsMap.values.any((String? element) => element == null)) {
       messageBuffer.writeln('\nYou must set up the missing settings.');
     }
+
+    if (xcodeProjectSettingsMap['Bundle Identifier']?.startsWith('com.example') ?? false) {
+      messageBuffer.writeln('\nWarning: You may want to update your bundle identifier.');
```

Comment:
I think "You may want to update your bundle identifier" is too vague and the old warning had more details.
```suggestion
      messageBuffer.writeln('\nWarning: Your application still contains the default 'com.example' signing identifier.');
```
Can you post an image of what it looks like when there are multiple of these kinds of warnings?  Is the double-spacing too much?

Reply:
Good point. Will update. i think i'll still call it "bundle identifier" instead of "signing identifier" since it matches with our website instruction. 

![Screen Shot 2022-12-02 at 11 22 27 AM](https://user-images.githubusercontent.com/41930132/205369676-d7aaa3d3-0668-43af-8065-2a58a30808a0.png)

Reply:
I can also do: 

```
Version Number: ...

Warning: 
- Your application still contains the default "com.example" bundle identifier. 
- App icon is using the wrong size (e.g. Icon-40.png).

To update the settings, please...
```

Reply:
This is probably better. Actually, let's discuss the redesign in a separate PR and I can loop in more people? (Just wanna make this PR atomic)

Reply:
The double spacing isn't as bad as I thought.  Let's keep it and we can circle back with UXR once you have landed all the IPA validation PRs.